misc/dbt-materialize/dbt/include/materialize/macros/adapters.sql

```diff
@@ -41,9 +41,30 @@
       -- See /relations/columns_spec_ddl.sql for details.
       {{ get_table_columns_and_constraints() }}
     {%- endif %}
+
   {% if cluster %}
     in cluster {{ cluster }}
   {% endif %}
+
+  {% set nullability_assertions = [] %}
```

Comment:
Do we need to be checking `contract_config.enforced`?

Reply:
I can add that in. Currently it will pick up column constraints without:
```
    config:
      contract:
        enforced: true
```
being set

Reply:
Adding that in will work but log `Materialize does not support constraints.` from [materialize__get_table_columns_and_constraints](https://github.com/MaterializeInc/materialize/blob/main/misc/dbt-materialize/dbt/include/materialize/macros/relations/columns_spec_ddl.sql).

Reply:
Right. We should do something more nuanced. I would have that warning only trigger if you're trying to apply _unsupported_ constraints.